### ISSUE TITLE
Sorting by file names in and among folders

### ISSUE THREAD

**Issue by Gouvernathor** [FossifyOrg/Music-Player#99]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Music-Player/issues) or [closed](https://github.com/FossifyOrg/Music-Player/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Music-Player/discussions) or [closed](https://github.com/FossifyOrg/Music-Player/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Music-Player/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

In the folder tab, you can sort by folder "title" (which is usually called a folder "name"), but when opening a folder, you can't sort the files by filename (which should even be the default when in folder mode), at best by song title which is very different and doesn't make real sense.
The sorting of folders by "title" should be renamed as sorting by "name", and there should be an option to sort by filename/filepath at least when opening a folder.

### Why do you want this feature?

Using the app in folder mode means that you want to use filenames and file paths as the basis for your music browsing.

**Comment by naveensingh** [comment]
Closing in favor of #393, which requests filename sorting as a standalone criterion not tied to folders or other settings.

**Comment by Gouvernathor** [comment]
@naveensingh I disagree : sorting by filename should be an option, sure, but in folder mode it should also be the default.
